src/pages/UserProfile/SelfProfile/SelfProfilePage.tsx

```diff
@@ -40,25 +30,25 @@ export default function SelfProfilePage(): JSX.Element {
       <div>
         <img
           className="h-32 w-full mx-auto object-cover lg:h-48"
-          src={userCoverImage}
+          src={userInfo?.coverImage}
           alt=""
         />
       </div>
       <div className=" -mt-12 mx-auto px-4">
         <div className="flex">
           <img
             className="h-24 w-24 rounded-full ring-4 ring-gray-100"
-            src={userProfileImage}
+            src={userInfo?.profileImage}
```

Comment:
lets use default profile image if this is null.

Reply:
//